catalog/task/create-pyxis-image/0.1/create-pyxis-image.yaml

```diff
@@ -0,0 +1,78 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: create-pyxis-image
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task that executes a python script to push a container image's metadata to Pyxis
+  params:
+    - name: pyxisUrl
+      type: string
+      description: The URL to the Pyxis API to be used
+    - name: pyxisSecret
+      type: string
+      description: The kubernetes secret to use to authenticate to Pyxis
+    - name: certified
+      type: string
+      description: If set to true, the image will be marked as certified in its Pyxis entry
+      default: "false"
+    - name: tag
+      type: string
+      description: The tag to use when pushing the container image metadata to Pyxis
+    - name: isLatest
+      type: string
+      description: If set to true, the image will have a latest tag added with its Pyxis entry
+      default: "false"
+    - name: skopeoInspectFile
+      type: string
+      description: The filename of the saved skopeo inspect output stored on the input workspace
+      default: "skopeo-inspect.json"
+  results:
+    - name: containerImageID
+      description: ID of the created entry in Pyxis
+  workspaces:
+    - name: input
+      description: The workspace where the skopeo-inspect.json file is located
+  steps:
+    - name: create-pyxis-image
+      image:
+        quay.io/hacbs-release/release-utils@sha256:1cb213e052adc0919f0ef1eb7f0fbb0f1a3b87cb98024095d1e1d036e8f5c5ac
+      env:
+        - name: pyxisCert
+          valueFrom:
+            secretKeyRef:
```

Comment:
I am not sure what SPI is going to do for us. I think it best to leave it as a secret like this for now, but we could change it in the future. The best would probably be to have the python script accept a variable instead of a file, but the script I took to start with had it like this

Reply:
OK, sounds good to leave it as is for now.